src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/package-info/test1/package-info.java

```diff
@@ -0,0 +1,5 @@
+/*OK: for test allowing to place annotation before PACKAGE_DEF.*/
+@Deprecated
+package com.puppycrawl.tools.checkstyle.checks.whitespace.test1.packageinfo;
```

Comment:
> com.puppycrawl.tools.checkstyle.checks.whitespace.test1.packageinfo

vs

> com/puppycrawl/tools/checkstyle/checks/whitespace/package-info/test1

path should be the same as package - "com/puppycrawl/tools/checkstyle/checks/whitespace/packageinfo/test1", or package the same path.
Please fix other cases too.